src/Kunstmaan/NodeBundle/Repository/NodeRepository.php

```diff
@@ -266,31 +267,30 @@ public function getAllMenuNodes(
         $connection = $this->_em->getConnection();
         $qb = $connection->createQueryBuilder();
         $databasePlatformName = $connection->getDatabasePlatform()->getName();
-        $createIfStatement = function (
+        $createIfStatement = static function (
             $expression,
             $trueValue,
             $falseValue
         ) use ($databasePlatformName) {
             switch ($databasePlatformName) {
                 case 'sqlite':
                     $statement = 'CASE WHEN %s THEN %s ELSE %s END';
-
                     break;
-
+                case 'postgresql':
+                    return sprintf('COALESCE(%s, %s)', $trueValue, $falseValue);
```

Comment:
```suggestion
                    return sprintf('COALESCE(%s, %s)', $falseValue, $trueValue);
```

As discussed on slack!